action.yml

```diff
@@ -0,0 +1,212 @@
+      id: comment-start
+      with:
+        comment-id: ${{ inputs.comment-id }}
+        issue-number: ${{ inputs.issue-number }}
+        body: |
+          > ${{ inputs.start-message || '🤖 **Starting Devin AI session...**' }}
+          >
+          > [Link to job logs.](${{ steps.vars.outputs.run-url }})
+          >
+
+    - name: Create Devin session
+      id: devin-session
+      shell: bash
+      env:
+        DEVIN_TOKEN: ${{ inputs.devin-token }}
+      run: |
+        set -euo pipefail
+        
+        echo "Creating Devin session..."
+        
+        # Prepare the API request payload
+        cat > devin_request.json << 'EOF'
+        {
+          "prompt": "${{ steps.build-prompt.outputs.prompt }}",
+          "repository_url": "https://github.com/${{ github.repository }}",
+          "branch": "${{ github.ref_name }}",
+          "metadata": {
+            "github_run_id": "${{ github.run_id }}",
+            "github_repository": "${{ github.repository }}",
+            "triggered_by": "github_action"
+          }
+        }
+        EOF
+        
+        echo "::group::📤 Devin API Request"
+        cat devin_request.json | jq
+        echo "::endgroup::"
+        
+        # Make API call to Devin
+        response=$(curl -s -X POST \
+          -H "Authorization: Bearer ${DEVIN_TOKEN}" \
+          -H "Content-Type: application/json" \
+          -d @devin_request.json \
+          "https://api.devin.ai/v1/sessions" || echo "API_CALL_FAILED")
+        
+        if [[ "$response" == "API_CALL_FAILED" ]]; then
+          echo "Failed to call Devin API"
+          exit 1
+        fi
+        
+        echo "::group::📥 Devin API Response"
+        echo "$response" | jq
+        echo "::endgroup::"
+        
+        # Extract session information
+        session_id=$(echo "$response" | jq -r '.session_id // .id // empty')
+        session_url=$(echo "$response" | jq -r '.session_url // .url // empty')
+        
+        if [[ -z "$session_id" ]]; then
+          echo "Failed to extract session ID from Devin API response"
+          exit 1
+        fi
+        
+        echo "Devin session created successfully!"
+        echo "Session ID: $session_id"
+        echo "Session URL: $session_url"
+        
+        echo "session-id=$session_id" >> $GITHUB_OUTPUT
+        echo "session-url=$session_url" >> $GITHUB_OUTPUT
+
+    - name: Append success comment
+      if: steps.comment-start.outputs.comment-id && success()
+      uses: peter-evans/create-or-update-comment@v4
+      with:
+        comment-id: ${{ steps.comment-start.outputs.comment-id }}
+        reactions: hooray
+        body: |
+          > ${{ inputs.success-message || '✅ **Devin AI session created successfully!**' }}
+          >
+          > 🔗 **Session Details:**
+          > - Session ID: `${{ steps.devin-session.outputs.session-id }}`
+          > - Session URL: ${{ steps.devin-session.outputs.session-url }}
+          >
```

Comment:
Here in the confirmation, I think it's sufficient to just append the URL and not also the session ID.